### PR TITLE
Add feature for just the urlencode builtin

### PR DESCRIPTION
**Pull request by msrd0**
This way, I don't have to pull in a bunch of other dependencies that I don't need, but can still have the urlencode filter which I do need.

### CODE REVIEW

**Comment by msrd0** [comment]
@Keats any chance you can take a look at this PR?

File: Cargo.toml [hunk 1]
```diff
@@ -43,7 +43,8 @@ tempfile = "3"
 
 [features]
 default = ["builtins"]
-builtins = ["slug", "percent-encoding", "humansize", "chrono", "chrono-tz", "rand"]
+builtins = ["builtins_urlencode", "slug", "humansize", "chrono", "chrono-tz", "rand"]
+builtins_urlencode = ["percent-encoding"]
```
Comment (Keats) [inline Cargo.toml:47]: can you just name the feature `urlencode`?

Comment (msrd0) [reply Cargo.toml]: yes, I renamed the feature to just `urlencode`

**Comment by Keats** [comment]
cool, I'll merge it with https://github.com/Keats/tera/pull/711 when both are ready

**Pull request merged**